### PR TITLE
docs: update docs to mention grafana helm chart

### PR DESCRIPTION
**Pull request by GabrielCastro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ For available configuration settings, please refer to [Grafana Image Rendering d
 
 Instead of installing and running the image renderer as a plugin, you can run it as a remote image rendering service using Docker. Read more about [remote rendering using Docker](https://github.com/grafana/grafana-image-renderer/blob/master/docs/remote_rendering_using_docker.md).
 
+### Remote Rendering in Kubernetes
+
+The grafana [helm chart](https://github.com/grafana/helm-charts/tree/main/charts/grafana) supports creating a remote renderer by setting `imageRenderer.enabled = true` settings are configured in the same way as [remote rendering using Docker](https://github.com/grafana/grafana-image-renderer/blob/master/docs/remote_rendering_using_docker.md).
```
Comment (aknuds1) [inline README.md:48]:
```suggestion
The Grafana [Helm chart](https://github.com/grafana/helm-charts/tree/main/charts/grafana) supports creating a remote renderer by setting `imageRenderer.enabled = true`. Settings are configured in the same way as [remote rendering using Docker](https://github.com/grafana/grafana-image-renderer/blob/master/docs/remote_rendering_using_docker.md).
```

File: README.md [hunk 2]
```diff
@@ -43,6 +43,10 @@ For available configuration settings, please refer to [Grafana Image Rendering d
 
 Instead of installing and running the image renderer as a plugin, you can run it as a remote image rendering service using Docker. Read more about [remote rendering using Docker](https://github.com/grafana/grafana-image-renderer/blob/master/docs/remote_rendering_using_docker.md).
 
+### Remote Rendering in Kubernetes
```
Comment (aknuds1) [inline README.md:46]: We use sentence case in [titles](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#headings).
```suggestion
### Remote rendering in Kubernetes
```

File: docs/remote_rendering_using_docker.md [hunk 3]
```diff
@@ -150,6 +150,16 @@ services:
 3. Create a dashboard, add a panel and save the dashboard.
 4. Panel context menu -> Share -> Direct link rendered image
 
+## Enable in grafana helm chart
```
Comment (aknuds1) [inline docs/remote_rendering_using_docker.md:153]:
```suggestion
## Enable in Grafana Helm chart
```

File: docs/remote_rendering_using_docker.md [hunk 4]
```diff
@@ -150,6 +150,16 @@ services:
...
+
+The [helm chart](https://github.com/grafana/helm-charts/tree/main/charts/grafana#image-renderer-plug-in) supports directly enabling a remote image renderer deployment.
```
Comment (aknuds1) [inline docs/remote_rendering_using_docker.md:155]:
```suggestion
The [Helm chart](https://github.com/grafana/helm-charts/tree/main/charts/grafana#image-renderer-plug-in) supports directly enabling a remote image renderer deployment.
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]
Just leaving suggestions for language fixes I spotted. 

@marefr Do you know if it makes sense for us to mention the Helm chart? Do we even support Helm?

**Comment by GabrielCastro** [comment]
It's supported here: https://github.com/grafana/helm-charts/tree/main/charts/grafana

**Comment by aknuds1** [comment]
Thanks very much for the PR, but we are sorry to inform you that we've decided to close it, since we think Helm specific information is better placed in the Helm chart documentation itself. We don't provide Helm support as part of this repo.

**Pull request closed without merging**